src/panels/meta.tsx

```diff
@@ -42,14 +39,13 @@ function getMeta(stack: Stack, item: Item, content: string): MetaValue[] {
       ),
     });
   }
+  */
 
-  if (item.stack_id && item.mime_type == "text/plain") {
-    const textMeta = getTextMeta(b64ToUtf8(content));
-
+  if (item.stack_id && content.mime_type == "text/plain") {
     const info = [
-      { s: "word", n: textMeta.words },
-      { s: "char", n: textMeta.chars },
-      { s: "token", n: item.tiktokens },
+      { s: "word", n: content.words },
```

Comment:
The beginnings of decoupling content meta calculations from the UI. This meta will be calculated async soon